final_task/calculator/calculation.py

```diff
@@ -0,0 +1,74 @@
+import calculator.errors as error
+import calculator.parser as pars
+from calculator.interface import importing
+from calculator.vars import OPERATORS, math_const
+
+arg1, arg2 = None, None
+
+
+def sorting(expression: list) -> list:
+    """ convert expression in reverse polish notation"""
+    buff = []
+    RPL = []
+    for element in expression:
+        if element in OPERATORS:
+            while buff and buff[-1] != "(" and OPERATORS[element][0] <= OPERATORS[buff[-1]][0] and element != '^':
+                RPL.append(buff.pop())
+            buff.append(element)
+        elif element == ")":
+            while buff:
+                oper = buff.pop()
+                if oper == "(":
+                    break
+                RPL.append(oper)
+        elif element == "(":
+            buff.append(element)
+        else:
+            RPL.append(element)
+    while buff:
+        RPL.append(buff.pop())
```

Comment:
Two lists can be joined with `.extend` method

Reply:
fixed